flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/table/descriptors/AbstractHBaseValidator.java

```diff
@@ -44,20 +43,26 @@ public void validate(DescriptorProperties properties) {
 		super.validate(properties);
 		properties.validateValue(CONNECTOR_TYPE, CONNECTOR_TYPE_VALUE_HBASE, false);
 		properties.validateString(CONNECTOR_TABLE_NAME, false, 1);
-		properties.validateString(CONNECTOR_ZK_QUORUM, false, 1);
+		properties.validateString(CONNECTOR_ZK_QUORUM, validateZkQuorum(), 1);
 		properties.validateString(CONNECTOR_ZK_NODE_PARENT, true, 1);
 		validateSinkProperties(properties);
 		validateVersion(properties);
 	}
 
+	protected abstract boolean validateZkQuorum();
```

Comment:
minor: How about renaming to `zkQuorumIsOptional()`?

Reply:
Done, thanks.